### DISCUSSION TITLE
Create collection entry with REST

### DISCUSSION THREAD

**Discussion by catalinux17** [pocketbase/pocketbase#188] [Q&A] [answered]
I want to create an entry into a collection using Postman or Insomnia.

![image](https://user-images.githubusercontent.com/49818736/180296841-2fb52018-3a37-48d2-9d5a-6ca21b5bc272.png)
 I copied the token and 
 
 
![image](https://user-images.githubusercontent.com/49818736/180296990-17c4b31e-9944-4b28-a525-a9f77b4336ce.png)

Any suggestion?

**Comment by ganigeorgiev** [comment, marked as answer]
You need to specify the token schema `User` or `Admin` (it seems that Insomnia uses `Bearer` by default), or in other words - the `Authorization` header value should be in the format `User YOUR_TOKEN` or `Admin YOUR_TOKEN` (see also https://pocketbase.io/docs/api-authentication/).

I'm not familiar with the Insomnia interface, but I see that there is a "Header" tab, so I would click on it to check what is the `Authorization` header that is added to the request.

_btw. You can also change your Collection permissions by editing the Collection's API rule for the specific action (list, view, create, update, delete) from the Admin UI._

**Reply by catalinux17** [reply]
Thanks. I'll give it a try right now.

**Reply by catalinux17** [reply]
working!!!

**Reply by TylerBarnes** [reply]
Did this change? Looks like I need to leave out "Admin" for the request to go through

**Reply by ganigeorgiev** [reply]
@TylerBarnes Yes, the API has changed slightly with v0.8+ (you can read more in https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#migrate-from-v07x).

The auth schema is no longer necessary since it is part of the token payload, aka. `Authorization: TOKEN` (you can also refer to the documentation - https://pocketbase.io/docs/authentication/).

**Reply by TylerBarnes** [reply]
Nice, thanks @ganigeorgiev !
